securetea/lib/log_monitor/server_log/detect/attacks/xss.py

```diff
@@ -51,6 +53,9 @@ def __init__(self, debug=False):
         # Logged IP list
         self.logged_IP = list()
 
+        # Initialize OSINT object
+        self.osint_obj = OSINT(debug=debug)
+        
```

Comment:
Some blank spaces are there in line 58. Please remove the line and add a blank line.